### ISSUE TITLE
Color picker: color pointer not visible

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#31645]
Refs: #31457

1. Notice that the color pointer in the color picker is not visible enough it is always a green hollow circle

I see that chrome uses a white cirlce which is by far more visible.

A more advanced solution would be to always use the invert color of the one you are hovering over, that way it would always be visible.
White is better but when you go over light colors it is also invisible which is an accessiblity problem

Also in the HC theme it should have stronger color than white

**Comment by michelkaporin** [comment]
Wondering why was it even green in the first place.. 😄
